services/order.py

```diff
@@ -0,0 +1,32 @@
+from django.db.models import QuerySet
+from datetime import datetime
+from django.db import transaction
+from db.models import Order, Ticket, User, MovieSession
+
+
+def create_order(tickets: list[dict], username: str, date: str = None) -> None:
+    with transaction.atomic():
+        user = User.objects.get(username=username)
+        order_tickets = Order(user=user)
+        order_tickets.save()
+        if date:
+            created_at_date = datetime.strptime(date, "%Y-%m-%d %H:%M")
+            order_tickets.created_at = created_at_date
+            order_tickets.save()
```

Comment:
Be cautious when manually setting the `created_at` field, as it is typically managed automatically by Django. Ensure this logic is necessary and does not conflict with the intended use of the `created_at` field.